group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -12722,15 +12722,10 @@ public void testConsumerGroupMemberUsingClassicProtocolFencedWhenSessionTimeout(
         MockCoordinatorTimer.ExpiredTimeout<Void, CoordinatorRecord> timeout = timeouts.get(0);
         assertEquals(consumerGroupSessionTimeoutKey(groupId, memberId), timeout.key);
         assertRecordsEquals(
-            Arrays.asList(
-                // The member is removed.
-                CoordinatorRecordHelpers.newCurrentAssignmentTombstoneRecord(groupId, memberId),
-                CoordinatorRecordHelpers.newTargetAssignmentTombstoneRecord(groupId, memberId),
-                CoordinatorRecordHelpers.newMemberSubscriptionTombstoneRecord(groupId, memberId),
-
-                // The group epoch is bumped.
-                CoordinatorRecordHelpers.newGroupEpochRecord(groupId, 11)
-            ),
+            Stream.concat(
```

Comment:
Personally, I prefer the previous way. It repeats things but it is easier to read.